packages/modes/mode.go

```diff
@@ -0,0 +1,23 @@
+package modes
+
+import (
+	"github.com/GenesisKernel/go-genesis/packages/conf"
+	"github.com/julienschmidt/httprouter"
+)
+
+type ModeType int
+
+const (
```

Comment:
Useless stuff, just create more NodeMode methods, if you need Mode dependend operations